cmd/server.go

```diff
@@ -98,6 +100,12 @@ func createServerCmd(execer fakeruntime.Execer, httpServer server.HTTPServer) (c
 
 	// gc related flags
 	flags.IntVarP(&opt.gcPercent, "gc-percent", "", 100, "The GC percent of Go")
+	//grpc_tls
+	dir,_:=os.Getwd()
+	cwd:=filepath.Dir(dir)
+	flags.StringVarP(&opt.tls,"tls-grpc","",os.Getenv("TLS_MODE"),"The tls mode, supported: tlsup. Keep it empty to disable tls")
+	flags.StringVarP(&opt.tlsCert, "cert-file", "", filepath.Join(cwd, "certs",""), "The path to the certificate file")
```

Comment:
IMO, the default value should be empty.

Reply:
I hope users can use relative paths to read the certificates, so I did it this way